sci-libs/libbraiding/libbraiding-1.3.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Expose the functionality of cbraid as a shared library"
+HOMEPAGE="https://github.com/miguelmarco/libbraiding"
+
+SRC_URI="https://github.com/miguelmarco/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+# A few source headers still say GPLv2, but I believe that to be an
+# oversight: https://github.com/jeanluct/cbraid/issues/4
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~riscv"
```

Comment:
Can you change this to `~amd64` please? (All new versions should start "unstable.")

If you have pkgcheck installed you can run `pkgcheck scan --commits` after committing and it will run some checks to catch things like this.

Another thing it complains about is the commit message, which should be of the form "cat/pkg: summary". For example, "sci-libs/libbraiding: add 1.3"

Reply:
Done.